homeassistant/components/flexmeasures/sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Definition of the sensors of the FlexMeasures integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfEnergy
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, SIGNAL_UPDATE_SCHEDULE
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up sensor."""
+    hass.data[DOMAIN]["schedule"] = {"schedule": [], "start": None}
+
+    async_add_entities([FlexMeasuresScheduleSensor()], True)
+
+
+class FlexMeasuresScheduleSensor(SensorEntity):
+    """Sensor to store the schedule created by FlexMeasures."""
+
+    _attr_device_class = SensorDeviceClass.ENERGY
+    _attr_native_unit_of_measurement = UnitOfEnergy.MEGA_WATT_HOUR
+
+    def __init__(self) -> None:
+        """Sensor to store the schedule created by FlexMeasures."""
+        self._attr_unique_id = "flexmeasures_schedule"
+
+    @property
+    def name(self) -> str:
+        """Sensor name."""
+        return "FlexMeasures Schedule"
+
+    @property
+    def native_value(self) -> float:
+        """Average power."""
+
+        commands = self.hass.data[DOMAIN]["schedule"]["schedule"]
```

Comment:
Seeing lots of strings thst should be consts

Reply:
Changed some of the strings to constants. I left some strings that I think make sense since it's a dict key. If they all need to be constants I could change it.. b552bbe1e811f3c05c66f2ee5b6b454484d83e08

Reply:
sorrry, this changed in another commit again which also contains some other changes. c5edcf87c7a1a20bf4b95b3fca4bd16a5505cf72